integ-tests/src/cucumberTest/resources/features/dataservice/list_environments.feature

```diff
@@ -1,26 +1,41 @@
-@ignore
 @dataservice
 @list-environments
 Feature: List environments
 
+  @ignore
   Scenario: List environments on a non-existent cluster
-    When I list environments on a cluster that does not exist
+    When I list environments in a cluster that does not exist
     Then 0 environments are returned
```

Comment:
If we change this line into "there environment are returned" and with empty return list, we can skip implementing this step, and can reuse the "there environment are returned" step.

Reply:
I think this scenario is testing listEnvrionments with a cluster that is not exist, in which case, the API will throw an exception. Since we haven't implement validation for the cluster, so I mark this scenario as ignore and will leave the detail implementation in future.

Reply:
Sure. Let's save this one for future implementations.